src/wp-includes/class-wp-script-modules.php

```diff
@@ -178,6 +178,10 @@ public function add_hooks() {
 		add_action( $position, array( $this, 'print_import_map' ) );
 		add_action( $position, array( $this, 'print_enqueued_script_modules' ) );
 		add_action( $position, array( $this, 'print_script_module_preloads' ) );
+
+		add_action( 'admin_footer', array( $this, 'print_import_map' ) );
+		add_action( 'admin_footer', array( $this, 'print_enqueued_script_modules' ) );
+		add_action( 'admin_footer', array( $this, 'print_script_module_preloads' ) );
```

Comment:
Just like in #6453, we can use `admin_print_footer_scripts` which is a bit later and probably more appropriate.